odoo/tools/translate.py

```diff
@@ -171,6 +171,7 @@ def translate_attrib_value(node):
 
 avoid_pattern = re.compile(r"\s*<!DOCTYPE", re.IGNORECASE | re.MULTILINE | re.UNICODE)
 node_pattern = re.compile(r"<[^>]*>(.*)</[^<]*>", re.DOTALL | re.MULTILINE | re.UNICODE)
+isspace_pattern = re.compile(r"[\s\uFEFF]*")  # web_editor uses \uFEFF as ZWNBSP
```

Comment:
I personally find the variable name a bit awkward.  Why not simply `space_pattern`?

Reply:
thought I'd name it similarly to the `isspace` method it replaces, but `space_pattern` works for me too